src/Components/CreateImageWizard/steps/Firewall/components/Services.tsx

```diff
@@ -47,6 +54,13 @@ const Services = () => {
           stepValidation={stepValidation}
           fieldName='disabledServices'
         />
+        <HelperText>
+          <HelperTextItem variant='default'>
+            Use firewalld service names (e.g., <code>ssh</code>) rather than
+            systemd unit names. List available services with{' '}
```

Comment:
maybe write: 
'To see all available services, run firewall-cmd --get-services'
instead of 
'List available services with{' '}<code>firewall-cmd --get-services</code>.'